integration/basic_test.go

```diff
@@ -629,3 +629,37 @@ func TestDebugError(t *testing.T) {
 		require.NoError(t, db.Client().Ping(ctx, nil), "other errors should not close connection")
 	})
 }
+
+func TestPing(t *testing.T) {
+	// Shouldn't be ran in parallel, as NonExistentDB test requires
+	// the current state of database list, which can be changed by other tests.
+
+	ctx, collection := setup.Setup(t)
+	db := collection.Database()
+
+	t.Run("Multiple", func(t *testing.T) {
+		for i := 0; i < 5; i++ {
+			err := db.RunCommand(ctx, bson.D{{"ping", int32(1)}}).Err()
+			assert.NoError(t, err)
+		}
+	})
+
+	t.Run("NonExistentDB", func(t *testing.T) {
+		expectedDatabases, err := db.Client().ListDatabases(ctx, bson.D{})
+		require.NoError(t, err)
+
+		dbName := "NonExistentDatabase"
+		for _, database := range expectedDatabases.Databases {
+			require.NotEqual(t, dbName, database.Name)
+		}
```

Comment:
Would `require.Contains(t, expectedDatabases.Databases, dbName)` work instead of this loop?

Reply:
@chilagrow yes, exactly! Thanks for clarification :)

Reply:
It wouldn't work as `expectedDatabases.Databases()` returns the slice of structs that specify more than the `Name` field. The only way here is to access the name by the struct ".Name` field

Reply:
ok, then let's keep the manual check!

Reply:
Actually, I realized that we can just use `ListDatabases` filter for that:
```
		expectedDatabases, err := db.Client().ListDatabases(ctx, bson.D{{"name", dbName}})
		require.NoError(t, err)
		require.Empty(t, expectedDatabases.Databases)
```